geoips/plugins/modules/readers/ssmi_binary.py

```diff
@@ -617,7 +623,7 @@ def fill_arrays():
     xarray_85ab.attrs["source_name"] = "ssmi"
     xarray_85ab.attrs["platform_name"] = satid
     xarray_85ab.attrs["data_provider"] = "DMSP"
-    xarray_85ab.attrs["source_file_names"] = [basename(fname)]
+    xarray_85ab.attrs["source_file_names"] = [os.path.basename(fname)]
```

Comment:
This is great that we're using explicit calls to os.path rather than just importing basename